src/github.com/matrix-org/dendrite/common/config/appservice.go

```diff
@@ -182,9 +199,9 @@ func checkErrors(config *Dendrite) error {
 			}
 		}
 	}
-	setupRegexps(config)
 
-	return nil
+	err = setupRegexps(config)
+	return err
```

Comment:
Why are we not just `return setupRegexps(config)`, and why do we need to name the return value?

Reply:
Derp, yes that really should be `return setupRegexps(config)`.